### DISCUSSION TITLE
size of group buttons | how to change?

### DISCUSSION THREAD

**Discussion by Symbiot78** [punxaphil/custom-sonos-card#194] [Q&A] [answered]
Hi

i have a layout-card with speaker & group.

i want to change the size of the 'button's for the group card, so I can do 25% 75% for the 2 cards.
But I am missing something and it doesn't look good.

can someone give me a hint.. I've tried the widthPercentage but it seems to only change the size of the card.. not the buttons? also.. when going to like widthPercentage: 10 it doesn't really get smaller from the 40 I'd set it to.

idea's ?

![image](https://github.com/johanfrick/custom-sonos-card/assets/3314249/d47ab1c9-83c4-40c8-885c-e2fa4a0454a4)

**Comment by punxaphil** [comment]
That is not supported, but you can always give [card_mod](https://github.com/thomasloven/lovelace-card-mod) a try.
```yaml
type: custom:sonos-card
card_mod:
  style:
    .: |
    sonos-player$ sonos-player-controls$: |
      ha-icon-button {
        --mdc-icon-button-size: 1.5rem !important;
        --mdc-icon-size: 1.5rem !important;
      }
```
[![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/9bebc1f6-6bda-4cbd-b596-50d3d2f712de)](https://www.buymeacoffee.com/punxaphil)

**Comment by Symbiot78** [comment]
Just for my understanding. What you've suggested above @johanfrick  -  this is just an example right?

sonos-player$ sonos-player-controls$ isn't the group player selector right?
( I tried.. it didn't work)

**Reply by punxaphil** [reply, marked as answer]
Sorry, I misread your text there. Something like this?
```
    sonos-groups$: |
      mwc-list {
        width: 10rem;
      }
```

**Comment by Symbiot78** [comment]
@johanfrick  -  you sir are a star!
i hate that I cannot figure out how to find those css attributes and how to use them :(
I've scoured developer tools and only found sonos-groups & sonos-group .. but nothing with mwc-list..
nor how you know that it's not sonos groups$ sonos-group$ ..

thank you.

**Reply by punxaphil** [reply]
Happy it worked!
